### ISSUE TITLE
Reopen closed tab not working

### ISSUE THREAD

**Issue by Ben3eeE** [atom/image-view#53]
### Prerequisites

* [X] Can you reproduce the problem in [safe mode](http://flight-manual.atom.io/hacking-atom/sections/debugging/#check-if-the-problem-shows-up-in-safe-mode)?
* [X] Are you running the [latest version of Atom](http://flight-manual.atom.io/hacking-atom/sections/debugging/#update-to-the-latest-version)?
* [X] Did you check the [debugging guide](flight-manual.atom.io/hacking-atom/sections/debugging/)?
* [X] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
* [X] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
* [X] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if your bug or enhancement is already reported?

For more information on how to write a good [bug report](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-bug-report) or [enhancement request](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-enhancement-suggestion), see the `CONTRIBUTING` guide.

### Description

Reopen closed tab is not working. For text files it reopens closed tab, for image files it opens tab like: %5Ckicy%5Ch%5CIMG_20130501_123454.jpg - H:\kivy - atom

### Steps to Reproduce

1. Open image in atom.
2. Close tab of image in atom.
3. Right click on the tab bar and reopen closed tab.

**Expected behavior:** Atom to reopen last closed tab.

**Actual behavior:** Useless %255C tab opens.

### Versions
1.6.X
Windows 7.

**Comment by 50Wliu** [comment]
What was the original file named?

**Comment by Ben3eeE** [comment]
File name does not matter, in the case I posted it was IMG_20130501_123454.jpg located in H:\kivy\h.
Example on a different computer different file:
%5CUsers%5Clineri%5CDesktop%5Cissues%5Cfile.PNG here i open file.PNG from issues folder on the desktop.

There is also output in the console I see now, see attached image:
![open_closed_tab](https://cloud.githubusercontent.com/assets/1058982/14353464/78152440-fcd9-11e5-850f-4987ea847377.PNG)

**Comment by Ben3eeE** [comment]
Tested latest image-view master and still a problem, noticed that every time you reopen file:\ is added in the front of the filename and all special characters changed.
See image where I did a few reopens on the file `double pending.png` located on the desktop. On the first reopen the space gets changed to %20 and file:\ is added to the start. Next reopen file:\ is added to the start and the %20 gets converted to %2520 and it keeps going.
![reopen a few times](https://cloud.githubusercontent.com/assets/1058982/15735772/8d3e4e1c-289b-11e6-94aa-a0e8581afff8.png)
